src/packages/recompose/__tests__/renderNothing-test.js

```diff
@@ -1,8 +1,11 @@
 import test from 'ava'
+import React from 'react'
 import { renderNothing } from '../'
+import { shallow } from 'enzyme'
 
 test('renderNothing returns a component that renders null', t => {
-  const nothing = renderNothing('div')
-  t.is(nothing(), null)
-  t.is(nothing.displayName, 'Nothing')
+  const Nothing = renderNothing('div')
+  const wrapper = shallow(<Nothing />)
+  t.is(wrapper.type(), null)
```

Comment:
enzyme way to check that Component renders null [docs](https://github.com/airbnb/enzyme/blob/4d1517ab01c01011d7e7d1b4e9ae4201cd23268d/docs/api/ShallowWrapper/type.md)